src/Command/CleanupStringsCommand.php

```diff
@@ -59,7 +65,8 @@ public function __construct(
         SessionManager $sessionManager,
         SessionObjectiveManager $sessionObjectiveManager,
         CourseObjectiveManager $courseObjectiveManager,
-        ProgramYearObjectiveManager $programYearObjectiveManager
+        ProgramYearObjectiveManager $programYearObjectiveManager,
+        HttpClientInterface $httpClient
```

Comment:
Please inject our `Fetch` service instead. Someday we're finally going to fix #1446 and it will be much easier if networking is all consolidated.

Reply:
disagree. i checked out `Fetch`, and it uses the same `HttpClientInterface` underneath the hood that i'm injecting here. what's more, it is coded to download a file - i just send a `HEAD` request to get a response without the payload to see if i'm getting a 200/OK from the given URL. that should be good enough.

Reply:
Ok. I accept this.